sklearn/utils/fixes.py

```diff
@@ -435,3 +435,10 @@ def _smallest_admissible_index_dtype(arrays=(), maxval=None, check_contents=Fals
                     return np.int64
 
     return np.int32
+
+
+# TODO: Remove when Scipy 1.12 is the minimum supported version
+if sp_version < parse_version("1.12"):
+    from ..externals._scipy.sparse.csgraph import laplacian  # type: ignore  # noqa
+else:
+    from scipy.sparse.csgraph import laplacian  # type: ignore  # noqa
```

Comment:
I think that you are seeking for `# pragma: no cover` to indicate to codecov to not complain if the line is not covered.

Reply:
Thanks, got confused with these magic comments.